test/functional/example_test.py

```diff
@@ -78,7 +78,10 @@ def set_test_params(self):
         """Override test parameters for your individual test.
 
         This method must be overridden and num_nodes must be explicitly set."""
-        self.setup_clean_chain = True
+        # By default the test loads a pre-mined chain of 200 blocks from cache.
+        # Set skip_chain_setup to True to skip this and start from the Genesis
+        # block.
+        self.skip_chain_setup = True
```

Comment:
```suggestion
        self.use_cached_chain = False
```

The double negative is a good point. What about this name?

Reply:
Sounds good to me, happy to change it to that if other reviewers prefer it. :)

Reply:
> Sounds good to me, happy to change it to that if other reviewers prefer it. :)


It would make me feel less guilty of picking the "clean chain" name in fac93497986b5f74716383bf26c78406253c625a, so I hope that other reviewers take that into consideration :sweat_smile: